pom.xml

```diff
@@ -436,6 +438,17 @@
             <groupId>io.minio</groupId>
             <artifactId>minio</artifactId>
             <version>${lib.minio.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.jetbrains.kotlin</groupId>
+                    <artifactId>kotlin-stdlib-jdk8</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>org.jetbrains.kotlin</groupId>
+            <artifactId>kotlin-stdlib-jdk8</artifactId>
+            <version>${lib.kotlin-stdlib.version}</version>
         </dependency>
```

Comment:
Can we solve this with `dependencyManagement` instead? https://github.com/DependencyTrack/hyades-apiserver/blob/de481cb724e1e60b7fe0fccaad957bf46c521690/pom.xml#L149-L158

Also please add a comment as to why this is necessary, to make it easier for future-us when it comes to removing it again.